VulkanHppGenerator.cpp

```diff
@@ -511,8 +513,10 @@ namespace VULKAN_HPP_NAMESPACE
 #include <vulkan/vulkan_handles.hpp>
 #include <vulkan/vulkan_structs.hpp>
 #include <vulkan/vulkan_funcs.hpp>
+#include <vulkan/vulkan_shared.hpp>
```

Comment:
As the SharedHandle is an addition to vulkan.hpp, I would not include it here, but instead include vulkan.hpp from vulkan_shared.hpp.
Whoever uses the SharedHandles would then need to include vulkan_shared.hpp.

Reply:
sure